frontend/src/types/contributor.ts

```diff
@@ -46,6 +46,7 @@ export interface PR {
   contributor: string
   description: string
   url: string
+  milestoneAlignment?: boolean
```

Comment:
make sure it's clear that booleans are booleans in your var names
```suggestion
  isAlignedWithMilestone?: boolean
```